### PR TITLE
[JS] Topic: Tutorial on Object Oriented Javascript(using functions)

### LINKED ISSUE #460: JS: Object-Oriented JavaScript (using Functions)
# Object-Oriented JavaScript (using Functions)
Discuss the OOP concept using functions in Javascript.

## Directory
Topics/Object_Oriented_Javascript_Using_Functions

## Description
- Talk about the OOP concept in general
- How Functions can be used in OOP
-  this keyword

## Formats
 - [x] Document
 - [ ] Video
 - [ ] Audio

### PR DESCRIPTION
**Pull request by arp99**
<hr>

## Description 📜
Wrote a tutorial on Object Oriented Javascript(using functions)

Fixes #460 
## References:
1. MDN Docs : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Inheritance_and_the_prototype_chain
2. Youtube : https://www.youtube.com/watch?v=OSjTmET11OQ&ab_channel=codeWithSimran
3. Freecodecamp : https://www.freecodecamp.org/news/how-javascript-implements-oop/
4. Medium : https://medium.com/@ryansolid/oop-with-functions-in-javascript-a3ec30e4750a
5. Simplilearn : https://www.simplilearn.com/tutorials/javascript-tutorial/oop-in-javascript 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

### CODE REVIEW

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

**Review by paramsiddharth — APPROVED** [review]

**Comment by s4-2** [comment]
@arp99 Your doc looks good to me. I'll suggest you to change the format of link in **Description --> Reference** Write as **1 [freecodecamp](https://www.freecodecamp.org/news/how-javascript-implements-oop/)** and so on. Hope you got it:)

**Comment by arp99** [comment]
> A very documented Markdown file!! Great job @arp99 🔥🔥🔥
> Add `__Contributor:__ [Arpan Mondal](https://www.github.com/arp99)` at the end of every MD file.

I have added a commit, please do check

File: JavaScript/Topics/11.OOP_Javascript_Using_Functions/Inheritence/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+const myPrototype = Object.getPrototypeOf(car1);
+console.log(myPrototype);
+```
+![getPrototypeOf](../images/Object%20Prototype.png)
+
+## Summary
+Inheritance in JavaScript is when an object is based on another object. Inheritance allows us to reuse existing code, having objects take on properties of other objects. That's it the end of discussion.
+
+> Author :
+> <cite>[Arpan Mondal](https://github.com/arp99)</cite>
```
Comment (dakshsethi) [inline JavaScript/Topics/11.OOP_Javascript_Using_Functions/Inheritence/README.md:121]: Update this also as i have mentioned.

**Review by dakshsethi — CHANGES_REQUESTED** [review]

File: JavaScript/Topics/11.OOP_Javascript_Using_Functions/Inheritence/README.md (hunk 1 above)
Comment (arp99) [reply JavaScript/Topics/11.OOP_Javascript_Using_Functions/Inheritence/README.md]: Done

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

**Review by Creatoon — APPROVED** [review]

**Review by paramsiddharth — APPROVED** [review]

**Pull request merged**